<launch>
    <node name="camera_sweep_action_server" pkg="com2009_actions" type="camera_sweep.py" output="screen">
    </node>
</launch>